src/include/visit-cmake.h.in

```diff
@@ -375,6 +375,6 @@ logic here to cause compilation failure if this logic seems to be failing.
 /* define control to intentionally leak things or not. my profiling shows it
    saves on the order of 0.001 sec to undefine this. leaving it undefined
    for now in case there's some issue with it when using a remote server.
-*/
```

Comment:
Can you add a comment that this shouldn't be checked in with DEBUG_MEMORY_LEAKS uncommented out?

Can you also capitalize the first word of each sentence in the comment since you're modifying it?

Reply:
I completely reworked the comment.